app/templates/main/browse.html

```diff
@@ -29,37 +29,36 @@
             </div>
         </div>
         {% if results %}
-            <div class="govuk-grid-column-full govuk-grid-column-full__browse-details">
```

Comment:
why has `<div class="govuk-grid-column-full govuk-grid-column-full__browse-details">` been moved inside of the div starting after line <!-- TABLE -->?

Reply:
removed extra div not required in browse.html for both desktop and mobile view